core/src/main/resources/org/apache/spark/ui/static/historypage.js

```diff
@@ -195,7 +195,7 @@ $(document).ready(function() {
         }
 
         $(selector).DataTable(conf);
-        $('#hisotry-summary [data-toggle="tooltip"]').tooltip();
+        $('#history-summary-table [data-toggle="tooltip"]').tooltip();
```

Comment:
The original is the spelling error.  I did not notice that I thought it was ID wrong. I will fix it and test it.